nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -236,6 +236,13 @@
      release announcement</link> for more information.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       <literal>PRETTY_NAME</literal> in <literal>/etc/os-release</literal>
+       uses the release now instead of full version
+       to be more suitable for presentation to the user
+     </para>
+   </listitem>
```

Comment:
Does this really need to be in the release notes?

Reply:
It has an effect on external tools (like ProtonDB reports), so it's good that users know why that has changed and is not a bug.